app/src/androidTest/java/com/hedvig/app/feature/payment/FreeMonthsCampaignTest.kt

```diff
@@ -60,7 +61,7 @@ class FreeMonthsCampaignTest : TestCase() {
                                 .fragments
                                 .monetaryAmountFragment
                                 .toMonetaryAmount()
-                                .format(context())
+                                .format(context(), Market.SE)
```

Comment:
I fear that this might be incorrect in cases where the `MarketManager.market` returns something different than `Market.SE`. Perhaps we could have a function that loads the currently loaded market from the `MarketManager`? Given that we're working with test code, it doesn't have to be super-pretty, we can probably even do something like this:

```kotlin
inline fun market() = getKoin().get<MarketManager>().market
```